ui/src/organizations/components/CollectorRow.tsx

```diff
@@ -17,6 +17,7 @@ interface Props {
   collector: Telegraf
   bucket: string
   onDownloadConfig: (telegrafID: string) => void
+  onDelete: (telegrafID: string) => void
 }
 
 export default class BucketRow extends PureComponent<Props> {
```

Comment:
I feel like this class name got left behind from the copy and paste? :)

Reply:
Oops